projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -429,6 +429,12 @@ public function add_options_menu() {
 		// would conflict with our own Settings > Performance that links to Calypso, so we hide it it since the Calypso
 		// performance settings already have a link to Page Optimize settings page.
 		$this->hide_submenu_page( 'options-general.php', 'page-optimize' );
+
+		// Hide Settings > Performance when the interface is set to wp-admin.
+		// This is due to these settings are mostly also available in Jetpack > Settings, in the Performance tab.
+		if ( get_option( 'wpcom_admin_interface' ) === 'wp-admin' ) {
+			$this->hide_submenu_page( 'options-general.php', 'https://wordpress.com/settings/performance/' . $this->domain );
```

Comment:
Would it be better to avoid adding the submenu page when the interface is set to wp-admin instead of adding the menu and then hiding it later?

Reply:
Hmmm... I think it's fine since class-admin-menu.php has the same pattern by hiding menu items from Core. This way, the override is still unidirectional (child class overriding parent class) instead of two ways. WDYT? 

Or are you suggesting moving the logic to class-admin-menu.php since it's possible that we want to the classic admin interface, no matter if it's Simple or Atomic site?

Reply:
Ok. We can consider this later when we want to apply the changes to simple sites 🙂

Reply:
I checked and there is no Jetpack -> Settings -> Performance in Simple sites, so yes we can revisit this later.